corehq/util/workbook_json/excel_importer.py

```diff
@@ -15,6 +15,7 @@ class ExcelImporter(object):
     def __init__(self, task, file_ref_id):
         self.task = task
         self.progress = 0
+        self.total_rows = 100
```

Comment:
That's so we can use the `add_progress()` method before we know the value of `total_rows`. ... We could use `DownloadBase.set_progress()` instead, but we're using 100 as a default value two lines down anyway, and I figured using `self.add_progress()` was a tiny bit cleaner.